spec.example.yaml

```diff
@@ -23,4 +23,10 @@ prometheus:
 workers:
   count: 6
   machineType: c4.large
-  appInstances: 180
\ No newline at end of file
+  appInstances: 180
+simulator: |
+  name: fake-webserver
+  high: 200
+  low: 100
+  interval: 10
```

Comment:
How about naming this "loadGenerator" instead of "simulator"?

Then a sub-section "scaler" for the properties specified here so we can add a separate "querier" section.
What seems like a good approach is having functionality for either be provided by the same container for update simplicity, but only have one of them ever running. Both are ran by just having two distinct deployments. That makes it easy to turn either on or off on demand.